### DISCUSSION TITLE
Is there a plan to support .NET 5.0 ?

### DISCUSSION THREAD

**Discussion by mattiasjback** [fluentassertions/fluentassertions#1518] [General]
Hi I've been using FluentAssertions for my last couple of projects and I love it.
However, I'm starting a new project targeting the latest .NET 5.0.
I sadly discovered that FluentAssertions is not up to date to support this new framework.

What is the roadmap for adding support to .NET 5.0 and beyond?

**Comment by dennisdoomen** [comment]
What do you mean with that? We're using FA in multiple .NET 5 projects. Are you missing some feature?

**Reply by mattiasjback** [reply]
Hi, hmm ok. I just assumed it was lack of support for the new framework that caused my error.
 
For example in my xUnit test when writing
```
var temp = 1;
temp.Should.Be(1);
```
I get a compiler error 
https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs0119?f1url=%3FappId%3Droslyn%26k%3Dk(CS0119)
<img width="637" alt="Screenshot 2021-04-07 at 17 33 39" src="https://user-images.githubusercontent.com/81409788/113893609-7b6d9f80-97c7-11eb-95b0-957d249b0d85.png">


I'm using
Visual Studio 2019 for Mac v 8.9.5 build 4
.NET SDK 5.0.201

**Reply by jnyrup** [reply]
`Should` -> `Should()`

**Reply by mattiasjback** [reply]
Thx I should probably call it the day when making these rookie mistakes. Thx for the input and your time.

**Comment by jnyrup** [comment]
To extend on Dennis' comment. 
A .net 5 project can consume several of the frameworks we target.
For e.g. v5.10.3 of Fluent Assertions a .net 5 project will pick the .net core 2.1 version of FluentAssertions.dll.

**Reply by KinsonDigital** [reply]
I would recommend listing NET 5.0 in the list of NET versions that are supported.  I did not notice that it was on the list.  :)

**Reply by dennisdoomen** [reply]
The listed versions are the minimum versions. Later versions are always supported.

**Reply by jnyrup** [reply]
@KinsonDigital Which list(s) are you referring to?
The lists on https://fluentassertions.com/ and https://www.nuget.org/packages/fluentassertions specifies what frameworks we _target_ and what test frameworks we _support_.
A newer framework will pick up _some_ older target.
But I might be biased in believing that is quite common knowledge?

Tip, you can use https://nugettoolsdev.azurewebsites.net/5.8.0/get-nearest-framework to see which nuget target will be picked.

**Reply by KinsonDigital** [reply]
Ah ok.  Makes perfect sense.  Thanks!!
